frontend/src/pages/CompareV2.tsx

```diff
@@ -541,25 +674,34 @@ function CompareV2(props: PageProps) {
             selectedTab={metricsTab}
             onSwitch={setMetricsTab}
           />
-          <div className={classes(padding(20, 'lrt'))}>
+          <div className={classes(padding(20, 'lrt'), css.outputsOverflow)}>
             {metricsTab === MetricsType.SCALAR_METRICS && <p>This is the Scalar Metrics tab.</p>}
             {metricsTab === MetricsType.CONFUSION_MATRIX && (
               <MetricsDropdown
                 filteredRunArtifacts={confusionMatrixArtifacts}
+                metricsTab={metricsTab}
                 metricsTabText={metricsTabText}
+                selectedArtifacts={selectedArtifacts}
+                setSelectedArtifacts={setSelectedArtifacts}
               />
```

Comment:
(For comment in MetricsDropdownProps): To pass only the data sub component needs , how about the following?

```
            {metricsTab === MetricsType.CONFUSION_MATRIX && (
              <MetricsDropdown
                filteredRunArtifacts={confusionMatrixArtifacts}
                metricsTabText={metricsTabText}
                selectedArtifacts={selectedArtifacts[metricsTab]}
                updateSelectedArtifacts={(newArtifacts) => {
                     selectedArtifacts[metricsTab] = newArtifacts;
                     setSelectedArtifacts(selectedArtifacts);
                }}
              />
            )}